theme.json

```diff
@@ -239,39 +274,49 @@
 		"elements": {
 			"h1": {
 				"typography": {
-					"fontSize": "3rem"
+          "fontFamily": "var(--wp--preset--font-family--league-lothic)",
```

Comment:
I believe that we can eliminate some of the code duplication that's happening with `fontFamily` by putting it in `styles.elements.heading`, so that it's shared across all headings.

Reply:
Fixed here 32d14c6c1c7c74116227c0dcc7204eb3b9e03ff2